wormhole-connect/src/hooks/useFetchTokenPrices.ts

```diff
@@ -0,0 +1,55 @@
+import { TOKENS } from 'config';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { TokenPrices, setPrices, setPricesError } from 'store/tokenPrices';
+import { sleep } from 'utils';
+
+const PRICES_FETCH_INTERVAL = 60000 * 5; // 5 mins
+
+export const useFetchTokenPrices = () => {
+  const dispatch = useDispatch();
+  const usdPrices: TokenPrices = {};
+  useEffect(() => {
+    let cancelled = false;
+    const fetchTokenPrices = async () => {
+      while (!cancelled) {
+        try {
+          // Make API call to fetch token prices
+          const idsList = [];
+          for (const [, tokenConfig] of Object.entries(TOKENS)) {
```

Comment:
it could be refactored to

```suggestion
Object
  .values(TOKENS)
  .filter(config => !!config.coinGeckoId)
  .map(({ coinGeckoId }) => coinGeckoId)
  .join(',');
```

and since TOKENS is constant I think it could computed ahead of time to save computation cycles

Reply:
Done!